### DISCUSSION TITLE
Specifying Chromedriver executable path using undetected mode

### DISCUSSION THREAD

**Discussion by jdholtz** [seleniumbase/SeleniumBase#2103] [Q&A] [answered]
Hello. This is a fantastic project and I am transitioning over to it from undetected chromedriver for my own project. The only part I'm having trouble with is that I don't see an option that is similar to undetected chromedriver's `driver_executable_path` while using the `Driver` class.

I've searched similar issues and discussions, but I only found the `binary_location` argument to specify the location of the browser. Is there an option for specifying the driver executable path or is that unnecessary as SB downloads the driver in its own location?

**Comment by mdmintz** [comment, marked as answer]
Hi @jdholtz, Thank you. SeleniumBase has it's own location for downloading drivers. Drivers are downloaded to the `seleniumbase/drivers/` folder, which is based on where seleniumbase was installed. Therefore, specifying the driver location is not necessary.

If using regular (non-UC) mode, then seleniumbase will first check the PATH for a matching driver before downloading one to the `seleniumbase/drivers/` folder.

If you want the driver version to be different from the browser version, seleniumbase allows for a mismatched pairing, and you can specify the driver version to use with ``--driver-version=VERSION``. (The new options were added recently in https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.17.12)

**Reply by jdholtz** [reply]
Great. Thanks!
